x/subscription/spec/03_messages.md

```diff
@@ -46,11 +49,24 @@ ConsumeGas( count_period(create_time, expiration_time, plan.cron_spec, plan.cron
 
 ## MsgStopSubscription
 
-Both the subscriber and the plan owner can stop a subscription.
+Subscriber stop a subscription.
 
 ```protobuf
 message MsgStopSubscription {
-  int32 plan_id,
-  Address subscriber,
+  string subscriber, // signer
+  int64 plan_id,
 }
 ```
+
+## MsgStopUserSubscription
+
+The plan owner stop user's subscription.
+
+```protobuf
+message MsgCreateSubscription {
```

Comment:
```suggestion
message MsgStopUserSubscription {
```
the message content should just have plan_owner + plan_id?

Reply:
it should be `plan_owner` + `subscription_id`, updated spec.